assignments/array-methods.js

```diff
@@ -57,22 +57,34 @@ const runners = [
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs. Combine both the first and last names and populate a new array called `fullNames`. This array will contain just strings.
-let fullNames = [];
+let fullNames = runners.forEach(function(currentValue, index, array){
+  console.log(currentValue.first_name +" "+ currentValue.last_name);
```

Comment:
best practice is to use string interpolation with backticks rather than concatenating strings.